src/renderer/components/Price.js

```diff
@@ -67,7 +46,20 @@ const Price = ({
   iconSize,
   showAllDigits = true,
   withIcon = true,
-}: Props) => {
+}: Props) {
+  const effectiveUnit = unit || from.units[0];
+  const value = BigNumber(10 ** effectiveUnit.magnitude);
+  const rawCounterValueCurrency = useSelector(counterValueCurrencySelector);
+  const counterValueCurrency = to || rawCounterValueCurrency;
+  const rawCounterValue = useCalculate({
+    from,
+    to: counterValueCurrency,
+    value: value.toNumber(),
```

Comment:
we don't need to use a BigNumber then?

Reply:
BigNumber value is passed to CounterValue below.

CounterValue component doesn't really need to accept BigNumber as value prop but it's a way it is currently. Some component passes account.amount: BigNumber to it. I need to understand when to use BigNumber and when not to.